README.adoc

```diff
@@ -78,8 +78,9 @@ MicroProfile Config combines configuration properties from multiple sources, eac
 
 A higher ordinal value means that the values taken from this ConfigSource will override values from ConfigSources with a lower ordinal value.
 
-There are three default configuration sources as following:
+There are four default configuration sources as following:
 
+* A `<variable name="…​" value="…​"/>` element in server.xml has a default ordinal of 500.
```

Comment:
Suggestion:
"A `<variable name="..." value="..."/>` element in the `server.xml` file has a default ordinal of 500."